src/manage_nsfs/nsfs_schema_utils.js

```diff
@@ -32,19 +32,57 @@ schema_utils.strictify(account_schema, {
 // NOTE - DO NOT strictify nsfs_config_schema
 // we might want to use it in the future for adding additional properties
 
+/**
+ * validate_bucket_schema validates a account object against the NC NSFS account schema
+ * @param {object} account
+ */
 function validate_account_schema(account) {
     const valid = ajv.validate(account_schema, account);
-    if (!valid) throw new RpcError('INVALID_SCHEMA', ajv.errors[0]?.message);
+    if (!valid && ajv.errors[0]) {
```

Comment:
what should we throw if !valid but ajv.errors[0] is undefined?

Reply:
I changed the code, I hope it is more clear.
As I mentioned, I don't understand in which case the schema will be invalid and there will be an error without a message, which was in the code before.